src/main/java/org/example/P20_sumOfDigits.java

```diff
@@ -0,0 +1,19 @@
+package org.example;
+
+import java.util.Scanner;
+
+public class P20_sumOfDigits {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter your number: ");/ 2314
```

Comment:
I think you have a compilation error here.

Reply:
solved